ACPs/77-reinventing-subnets/README.md

```diff
@@ -230,36 +237,36 @@ type SubnetValidator struct {
     DisableOwner PChainOwner `json:"disableOwner"`
 }
 
-type ConvertSubnetTx struct {
+type ConvertSubnetToL1Tx struct {
     // Metadata, inputs and outputs
     BaseTx
     // ID of the Subnet to transform
     // Restrictions:
     // - Must not be the Primary Network ID
     Subnet ids.ID `json:"subnetID"`
```

Comment:
Is there a reason this param is `Subnet` and not `SubnetID` ? @StephenButtolph

Reply:
We can change it to `SubnetID`. It being named `Subnet` is an implementation detail of avalanchego.